src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,144 @@
+---
+path: /integrations/compass
+title: Compass
+description: This topic explains visualize LaunchDarkly feature flag changes in Atlassian Compass.
+published: true
+---
+
+<Callout intent="primary">
+<CalloutTitle>The Compass integration is a Pro and Enterprise feature</CalloutTitle>
+<CalloutDescription>
+
+The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+
+</CalloutDescription>
+</Callout>
```

Comment:
```suggestion
<Callout intent="primary">
  <CalloutTitle>The Compass integration is a Pro and Enterprise feature</CalloutTitle>
  <CalloutDescription>

  The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).

  </CalloutDescription>
</Callout>
```
The callout is a custom component we built, and it's pretty finicky about whitespace. This edit, and the similar ones below, just ensure the component builds and displays as intended. Here's a screenshot from my local build of your branch to confirm: 

<img width="620" alt="callouts after whitespace edits" src="https://user-images.githubusercontent.com/1653043/161335817-852f3cbc-39d2-4d93-ad30-9336fa1d4773.png">